day_two_guide/topics/removing_etcd_host.adoc

```diff
@@ -4,21 +4,39 @@ Removing an etcd host
 Module included in the following assemblies:
 
 * day_two_guide/host_level_tasks.adoc
+* admin_guide/assembly_restore-etcd-quorum.adoc
 ////
 
 If an etcd host fails beyond restoration, remove it from the cluster.
+ifeval::["{context}" == "restore-etcd-quorum"]
+To recover from an etcd quorum loss, you must also remove all healthy etcd
+nodes but one from your cluster.
+endif::[]
 
+ifeval::["{context}" == "day_two_host_level_tasks"]
 [IMPORTANT]
 ====
 Ensure the etcd cluster maintains quorum while removing the etcd host by
 removing a single host at a time from a cluster.
 ====
+endif::[]
 
 *Steps to be performed on all masters hosts*
 
 [discrete]
 == Procedure
```

Comment:
Do we need the _Procedure_ headings? To me they interrupt the flow and look odd when they follow a similar-size or smaller heading. Also, they seem redundant. It is a numbered list; of course it's a procedure! ;-)

Reply:
I agree with you, but the procedure template makes them seem necessary - https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc :(